src/agentNet/agent_net.py

```diff
@@ -0,0 +1,83 @@
+import networkx as nx
+import random
+import matplotlib.pyplot as plt
+import yaml
+from network_utils import *
+
+class AgentNet(nx.Graph):
```

Comment:
proponuję dodać informację czym są konkretne pola, zwłaszcza te mniej intuicyjne - na przykład odnoszę wrażenie że agent_policies to kolekcja mapująca id agentów na nazwy plików z ich konfigami, ale wymaga to wczytania się w kod klasy